metagraph/tests/graph/test_aligner.cpp

```diff
@@ -1451,6 +1451,13 @@ TEST(DBGAlignerTest, align_suffix_seed_snp_min_seed_length) {
         check_extend(graph, aligner.get_config(), paths, query);
     }
     {
+        size_t k = 15;
+        std::string reference = "TTTCGAGGCCAAAG" "CTTTCGAGGCCAA";
+        std::string query =                 "AC" "CTTTCGAGGCCAA";
```

Comment:
The point is to emphasize the position at which the two strings differ. Another way to do it would be to put a comment right above with an arrow indicating that position.

Reply:
I've annotated most of the sequences to mark the positions where edits occur.